### PR TITLE
Ct/fix set error

### LINKED ISSUE #4: Test failures (old redis-server?)
On some systems make test fails with a line
```
ERR wrong number of arguments for 'set' command# Tests were run but no plan was declared and done_testing() was not seen.
```
A sample fail report can be seen here: http://www.cpantesters.org/cpan/report/ae56db2e-b4dd-11e4-8dff-08b1e0bfc7aa

Maybe this is related to an older version of redis-server? I observe the failures on systems with the following redis-server version:
* Redis server version 2.4.14 (debian/wheezy)

And no failures with these versions:
* Redis server v=2.8.17 (debian/jessie)
* Redis server v=2.8.19 (freebsd 9 & 10)

### PR DESCRIPTION
**Pull request by einhverfr**
Attempt to fix #4 and provide diagnostics on #5

### CODE REVIEW

File: t/cache.t [hunk 1]
```diff
@@ -26,6 +27,11 @@ can_ok($cache, 'flushall');
 my $cache2 = Cache::RedisDB->redis;
 is $cache2, $cache, "Got the same cache object";
 
+my @version = split(/\./, $cache->info->{redis_version});
+my $sufficient_version = 1 if (($version[0] >= 2) && ($version[1] >= 6) && 
+                               ($version[2] >- 12));
```
Comment (mwmiller) [inline t/cache.t]: Is `>-` a typo or an operator with which I am not familiar?

**Comment by mwmiller** [comment]
Thanks for working on this, @einhverfr.  I had only considered ways to try to declare minimum versions, not work around them.  Nice!

File: t/cache.t (hunk 1 above)
Comment (einhverfr) [inline t/cache.t]: fixed.  Thanks!

File: t/cache.t [hunk 2]
```diff
@@ -7,6 +7,7 @@ use DateTime;
 use JSON qw(from_json);
 use RedisServer;
 use Cache::RedisDB;
+use Data::Dumper;
```
Comment (mm-binary) [inline t/cache.t]: Do you still need this line?

Comment (einhverfr) [inline t/cache.t]: fixed

**Pull request merged**